webapp/tests/unit/protected/histories/index/controller-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:protected/histories/index', 'Unit | Controller | protected/histories/index', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+// Replace this with your real tests.
```

Comment:
This is auto generated code by `ember generate`. There is nothing to be updated here but writing a full test which is not mandatory for PR to be accepted yet.